### ISSUE TITLE
[FEATURE] Scaffold PrioAura

### ISSUE THREAD

**Issue by Tcm9** [CCBlueX/LiquidBounce1.8-Issues#3207]
**Describe your feature request.**
If I have the setting PrioAura on, then Scaffold should not build any more if I have switched on the Killaura.

If you use Scaffold, and a person comes from behind, then I switch on Killaura quickly, but then I still hang on the corner, so that he doesn't hit right, but also doesn't build right. And if I also switch off Scaffold, then I could fall down.

If I also play Bedwars, then I can use scaffold as a savewalk without him building when I have switched on the Killaura. 

PrioAura is an abbreviation of Priority Killaura.
if you want, you can call it something else.

**Comment by Teals53** [comment]
keeprotations ?

**Comment by Tcm9** [comment]
No, it would help if Scaffold just doesn't build when the Killaura is on.  (setting)

**Comment by Tcm9** [comment]
In summary, if the setting proaura is turned on, and then the killaura is turned on, then as long as killaura is on, scaffold should become savewalk.

If sprint is switched off at scaffold, then it should be possible to sprint again while the killaura is on.
